com.github.hneemann.digital/com.github.hneemann.digital.json

```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "com.github.hneemann.digital",
```

Comment:
com.github app ID won't work. Did you see https://docs.flathub.org/docs/for-app-authors/requirements#application-id ? (It's linked in PR checklist)